curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-011.md

```diff
@@ -11,10 +11,22 @@ Up until now, you have been limited regarding the presentation and appearance of
 
 # --hints--
 
-Test 1
+Your code should have an opening `<style>` tag.
 
 ```js
+assert(code.match(/<style\s*>/i));
+```
+
+Your code should have a closing `</style>` tag.
+
+```js
+assert(code.match(/<\/style\s*>/));
+```
+
+Your `style` element should be nested in your `head` element.
```

Comment:
This is mostly an _out-of-interest_ question: Why have the editable region be more than just the `head` element?